directives/ae_output.py

```diff
@@ -109,6 +109,8 @@ def run(self):
             'category': 'active elements',
             'max_submissions': self.options.get('submissions', data.get('max_submissions', env.config.ae_default_submissions)),
         })
+        data["category"] = "active elements" if data["max_submissions"] == 0 else "active elements with max submissions"
```

Comment:
Looks like it wasn't possible to change the category of the active element in the RST directive options. But we could now add the possibility. If somebody wants to change the category, then we also do not need to set default setting (`accept_unofficial_submits`) to the new category.

Do you think this is OK?

Reply:
Yeah, I don't think we want to set `accept_unofficial_submits` on an user specified caregory as we don't really know what the intention is and what else is in that same category. I'll add the possibility of changing the category.